README.md

```diff
@@ -50,112 +49,158 @@ This allows our [`dojo cli`](https://github.com/dojo/cli) build tooling to make
 
 - All widgets are designed to be accessible. If custom ARIA semantics are required, widgets have an `aria` property that may be passed an object with custom `aria-*` attributes.
 
-- All widgets are fully themeable. Example themes are available in the [@dojo/themes](https://github.com/dojo/themes) repository.
+- All widgets are fully themeable. Example themes are available in the [/theme](https://github.com/dojo/widgets/tree/master/src/theme) folder.
 
 - All widgets support internationalization (`i18n`)
 
-## Widgets
+## Examples
 
-Live examples of current widgets are available in the [widget showcase](https://dojo.github.io/examples/widget-showcase/).
+Live examples of current widgets are available at [widgets.dojo.io](https://widgets.dojo.io ).
 
-### Form widgets
-[Button](src/button/README.md)
+## Writing widgets
 
-[Calendar](src/calendar/README.md)
+### Properties
+- defining properties
+- passing properties
+- reading properties
 
-[Checkbox/Toggle](src/checkbox/README.md)
+### Event callbacks
+- defining callbacks
+- passing callbacks
+- using event callbacks
 
-[ComboBox](src/combobox/README.md)
+### Control patterns
```

Comment:
This new section is super useful to everyone, not just consumers. What if this were moved up nearer the top to give consumers a view of the philosophy of how widgets have been developed.